### PR TITLE
Repeatable challenges

### PR DESCRIPTION
**Pull request by denkv**

### CODE REVIEW

File: src/main/java/org/hobbit/utils/rdf/RdfHelper.java [hunk 1]
```diff
@@ -171,6 +173,38 @@ protected static Calendar getCalendarValue(Model model, Resource subject, Proper
...
+        if (model == null) {
+            return null;
+        }
+        Literal literal = getLiteral(model, subject, predicate);
+        if (literal != null) {
+            try {
+                return Duration.parse(literal.getString());
+            } catch (Exception e) {
+                // nothing to do
+                LOGGER.debug("Couldn't parse \"" + literal.getString() + "\" as xsd:duration. Returning null.", e);
```
Comment (yamalight) [inline src/main/java/org/hobbit/utils/rdf/RdfHelper.java:201]: this bit won't return - is that intentional?

Comment (denkv) [reply src/main/java/org/hobbit/utils/rdf/RdfHelper.java:201]: There's return at the end of the function after this, and also existing `getDateTimeValue` has the same structure.

Comment (yamalight) [reply src/main/java/org/hobbit/utils/rdf/RdfHelper.java:201]: Ah, makes sense.

**Review by yamalight — APPROVED** [review]
LGTM

**Pull request merged**